boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -77,6 +77,11 @@ choice SPI_NOR_BLOCK_ERASE_SIZE
 	default SPI_NOR_BLOCK_ERASE_64K
 endchoice
 
+if BOOTLOADER_MCUBOOT
+config TEXT_SECTION_OFFSET
+	default 0x100
```

Comment:
I assume you want to save some space because `TEXT_SECTION_OFFSET` is set to 0x200 when `BOOTLOADER_MCUBOOT` is enabled.

Reply:
Yes. That's correct. There were some unused bytes in the text section but allocating 0x200 was overshooting it.

Reply:
Thanks for clarifying